runtime/src/bank/builtins/mod.rs

```diff
@@ -114,6 +114,13 @@ pub static BUILTINS: &[BuiltinPrototype] = &[
         program_id: solana_sdk::loader_v4::id(),
         entrypoint: solana_loader_v4_program::Entrypoint::vm,
     }),
+    testable_prototype!(BuiltinPrototype {
+        core_bpf_migration_config: None,
+        name: zk_elgamal_proof_program,
+        enable_feature_id: Some(feature_set::zk_elgamal_proof_program_enabled::id()),
+        program_id: solana_zk_sdk::elgamal_program::id(),
```

Comment:
nit for another PR: i'm just realizing that this is called `elgamal_program` -- would it be better to call it `elgamal_proof_program` to align with the other definitions?

Reply:
Agreed on above considering the name here is also defined as "zk_elgamal_proof_program".

Reply:
Okay yep, I created a PR to replace to update `elgamal_program` to `elgamal_proof_program` (https://github.com/anza-xyz/agave/pull/1698)